main.go

```diff
@@ -21,12 +22,17 @@ var (
 	maxResults = 200
 
 	// Command-line flags
-	apikey      string
-	appkey      string
-	doDashboard bool
-	doMonitor   bool
-	doService   bool
-	query       string
+	flagApiKey    bool
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [var flagApiKey should be flagAPIKey](https://app.codacy.com/manual/nekottyo/alfred-datadog-workflow/pullRequest?prid=5652591)